aspnet/mvc/views/tag-helpers/authoring/sample/AuthoringTagHelpers/src/AuthoringTagHelpers/Views/_ViewImports.cshtml

```diff
@@ -1,11 +1,11 @@
 ﻿@using AuthoringTagHelpers
 @addTagHelper *, Microsoft.AspNetCore.Mvc.TagHelpers
-@addTagHelper "AuthoringTagHelpers.TagHelpers3.EmailTagHelper, AuthoringTagHelpers"
+@addTagHelper "AuthoringTagHelpers.TagHelpers.EmailTagHelper, AuthoringTagHelpers"
 @addTagHelper "AuthoringTagHelpers.TagHelpers.WebsiteInformationTagHelper, AuthoringTagHelpers"
 @addTagHelper "AuthoringTagHelpers.TagHelpers.ConditionTagHelper, AuthoringTagHelpers"
 @addTagHelper "AuthoringTagHelpers.TagHelpers.AutoLinkerHttpTagHelper, AuthoringTagHelpers"
 @addTagHelper "AuthoringTagHelpers.TagHelpers.AutoLinkerWwwTagHelper, AuthoringTagHelpers"
-@addTagHelper "AuthoringTagHelpers.TagHelpers3.BoldTagHelper, AuthoringTagHelpers"
+@addTagHelper "AuthoringTagHelpers.TagHelpers.BoldTagHelper, AuthoringTagHelpers"
```

Comment:
I'll need to test these changes. I had to change the NS to avoid collisions. Does the code work with these changes?